app/src/main/java/com/github/se/cyrcle/ui/theme/molecules/Filter.kt

```diff
@@ -112,7 +115,7 @@ fun FilterPanel(
   val isTextFieldVisible = remember { mutableStateOf(false) }
 
   // Text field value
-  val textFieldValue = remember { mutableStateOf("") }
+  val textFieldValue = rememberSaveable { mutableStateOf("") }
```

Comment:
Why is the remember changed to a rememberSaveable?

Reply:
Note that the chosen location is maintained as it's part of the viewmodel. Julien decided it would be nice to have the Search Bar display the selected location. I just made sure to enforce this behaviour in the case i mentionned above

Reply:
This makes sense to me